i18n/ja/docusaurus-plugin-content-docs/current/guides/integration-mobile-client-server.md

```diff
@@ -0,0 +1,361 @@
+
+<Tabs groupId="language-selection">
+<TabItem value='android' label='Android'>
+
+```js
+UID2Manager.getInstance().generateIdentity()
+```
+
+</TabItem>
+<TabItem value='ios' label='iOS'>
+
+```js
+UID2Manager.shared.setIdentity()
+```
+
+</TabItem>
+</Tabs>
+
+## Token Storage
+
+After you call the `setIdentity` method, the UID2 Identity is persisted in local file storage.
+
+:::warning
+The format of the file stored in the local file storage, or the filename itself, could change without notice. We recommend that you do not read and update the file directly.
+:::
+
+## Pass Generated Token for Bid Stream Use
+
+(**GWH__KL02 Question from SW: "his is a question for PM probably - bid stream does not exist in UID2 Glossary - should it? I dunno if bid stream actually makes sense to publishers to be honest" Do we add it into glossary... I'd thought everyone would know that term, SW doesn't think so. And, bid stream two words... is that a company decision?**)
+
+To retrieve the token, in your mobile app, call the following:
+
+<Tabs groupId="language-selection">
+<TabItem value='android' label='Android'>
+
+```js
+UID2Manager.getInstance().getAdvertisingToken()
+```
+
+</TabItem>
+<TabItem value='ios' label='iOS'>
+
+```js
+UID2Manager.shared.getAdvertisingToken()
+```
+
+</TabItem>
+</Tabs>
+
+If a successful identity was added into the UID2Manager, this method returns a string such as the following:
+
+```
+AgAAAQFt3aNLXKXEyWS8Tpezcymk1Acv3n+ClOHLdAgqR0kt0Y+pQWSOVaW0tsKZI4FOv9K/rZH9+c4lpm2DBpmFJqjdF6FAaAzva5vxDIX/67UOspsYtiwxH73zU7Fj8PhVf1JcpsxUHRHzuk3vHF+ODrM13A8NAVlO1p0Wkb+cccIIhQ==
+```
+
+You can use this identity to pass downstream for sending in the RTB bid stream.
+
+If the `getAdvertisingToken()` method call returns `null`, there was no identity or valid token generated. Some possible reasons for this, and some things you could do to troubleshoot, are as follows:
+
+- The identity is invalid. Check to see whether there are any errors from the previous `setIdentity()` call.
+- You could enable logging to get more information: see [Enable Logging](#enable-logging).
+- The advertising token inside the UID2 identity has expired, and the refresh token has also expired, so the SDK cannot refresh the token.
+
+If there is no identity, follow the instructions in [Implement Server-Side Token Generation](#implement-server-side-token-generation) again, generate a new identity, and pass the result into your mobile app's UID2Manager again.
+
+## When to Pass a New UID2 Identity into the SDK
+
+The best way to determine whether a new UID2 identity is required by the UID2 SDK again is to call the `getAdvertisingToken()` method in all cases:
+
+<Tabs groupId="language-selection">
+<TabItem value='android' label='Android'>
+
+UID2Manager.getInstance().getAdvertisingToken()
```

Comment:
the code rendering doesn't look right here

Reply:
Fixed thx.